x-pack/plugins/ml/public/explorer/explorer_controller.js

```diff
@@ -264,7 +264,8 @@ module.controller('MlExplorerController', function (
 
     // Populate the map of jobs / detectors / field formatters for the selected IDs.
     mlFieldFormatService.populateFormats(selectedIds, getIndexPatterns())
-      .finally(() => {
+      .catch((err) => { console.log('Error setting selected jobs:', err); })
```

Comment:
Can you change this error message to say `Error populating field formats`. The jobs will be selected ok, it's just `actual` and `typical` values will be formatted using the default numeric formatting rather than e.g. as bytes, if a formatter has been applied for that field.